Sorry, I left a pep8 error in my PR:

stock_quant_merge/models/stock_move.py:5:1: F401 'odoo.api' imported but unused

Then travis will be green and we can merge

--- a/stock_quant_merge/models/stock_move.py
+++ b/stock_quant_merge/models/stock_move.py
@@ -2,7 +2,7 @@
 # © 2017 Eficent Business and IT Consulting Services S.L.
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
-from odoo import api, models
+from odoo import models
 
 
 class StockMove(models.Model):

